internal/vendor/wiz.go

```diff
@@ -339,11 +366,11 @@ type Project struct {
 
 // ProjectCloudAccountLink struct
 type ProjectCloudAccountLink struct {
-	CloudAccount   CloudAccount  `json:"CloudAccount"`
-	Environment    string        `json:"environment"` // enum Environment
-	ResourceGroups []string      `json:"resourceGroups"`
-	ResourceTags   []ResourceTag `json:"ResourceTag"`
-	Shared         bool          `json:"shared"`
+	CloudAccount   CloudAccount   `json:"cloudAccount"`
+	Environment    string         `json:"environment"` // enum Environment
+	ResourceGroups []string       `json:"resourceGroups"`
```

Comment:
This field is optional in the GraphQL schema.

```suggestion
	ResourceGroups []string       `json:"resourceGroups,omitempty"`
```

If there is a need to deviate from the GraphQL schema, please make a comment in the strct comments.